README.md

```diff
@@ -74,97 +79,92 @@ df.columns = ['x', 'y', 'z1']
 df['x2'] = df.x * df.x
 ```
 
+
 ## Documentation
 
 Project docs are published here: https://koalas.readthedocs.io
 
+
 ## Mailing List
 
 We use Google Groups for mailling list: https://groups.google.com/forum/#!forum/koalas-dev
 
 
 ## Development Guide
 
-### Environment Setup
+See [CONTRIBUTING.md](https://github.com/databricks/koalas/master/CONTRIBUTING.md).
 
-We recommend setting up a Conda environment for development:
-```bash
-conda create --name koalas-dev-env python=3.6
-conda activate koalas-dev-env
-conda install -c conda-forge pyspark=2.4
-conda install -c conda-forge --yes --file requirements-dev.txt
-pip install -e .  # installs koalas from current checkout
-```
 
-Once setup, make sure you switch to `koalas-dev-env` before development:
-```bash
-conda activate koalas-dev-env
-```
+## Design Principles
 
-### Running Tests
+This section outlines design princples guiding the Koalas project.
 
-There is a script `./dev/pytest` which is exactly same as `pytest` but with some default settings to run Koalas tests easily.
+### Be Pythonic
 
-To run all the tests, similar to our CI pipeline:
-```bash
-# Run all unittest and doctest
-./dev/pytest
-```
+Koalas targets Python data scientists. We want to stick to the convention that users are already familiar with as much as possible. Here are some examples:
 
-To run a specific test file:
-```bash
-# Run unittest
-./dev/pytest -k test_dataframe.py
+- Function names and parameters use snake_case, rather than CamelCase. This is different from PySpark's design. For example, Koalas has `to_pandas()`, whereas PySpark has `toPandas()` for converting a DataFrame into a pandas DataFrame. In limited cases, to maintain compatibility with Spark, we also provide Spark's variant as an alias.
 
-# Run doctest
-./dev/pytest -k series.py --doctest-modules databricks
-```
+- Koalas respects to the largest extent the conventions of the Python numerical ecosystem, and allows the use of numpy types, etc. that are supported by Spark.
```

Comment:
"that are supported by Spark.". I don't think officially PySpark supports NumPy types tho .. Maybe I am too worried but how about say, instead, "that can be supported by Spark"?

Reply:
sounds good